docs/schema.md

```diff
@@ -373,9 +372,11 @@ an explicitly enumerated `versions` list, because `GIT` range inclusion queries
 cannot be answered without access to a copy of the underlying Git repository.
 
 Again, it is important to note that to allow portable (non-ecosystem-specific)
```

Comment:
I think let's just remove this entire paragraph, because the point it's making here is no longer as strong.